src/ui/qgsrelationeditorconfigwidgetbase.ui

```diff
@@ -64,6 +64,36 @@
         </property>
        </widget>
       </item>
+      <item>
+       <layout class="QHBoxLayout" name="horizontalLayout">
+        <item>
+         <spacer name="horizontalSpacer">
+          <property name="orientation">
+           <enum>Qt::Horizontal</enum>
+          </property>
+          <property name="sizeType">
+           <enum>QSizePolicy::Fixed</enum>
+          </property>
+          <property name="sizeHint" stdset="0">
+           <size>
+            <width>10</width>
+            <height>20</height>
+           </size>
+          </property>
+         </spacer>
+        </item>
+        <item>
+         <widget class="QCheckBox" name="mAllowAddChildFeatureWithNoGeometry">
+          <property name="toolTip">
+           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked it is possible to add a feature whitout geometry to a spatial layer. Two &amp;quot;Add child feature&amp;quot; buttons are showed with and without geometry.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
```suggestion
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, it is possible to add a feature without geometry to a spatial layer. Two &amp;quot;Add child feature&amp;quot; buttons are showed : with and without geometry.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

One typo fixed
For punctuation, feel free to adapt or rephrase.

Reply:
Thank's, I will apply as you suggested, i just think the whitespace before the `:` is not needed right?

Reply:
True, in english we don't. (we do in French, that's why, sorry :) ). I have updated the suggestion.

Reply:
Oh I can write in french but didn't know about that! Learned something new thanks :)